src/sentry/event_manager.py

```diff
@@ -759,6 +759,7 @@ def save(self, project, raw=False):
         else:
             event_saved.send_robust(
                 project=project,
+                event_size=event.size,
```

Comment:
need this to track total volume in some related tasks

Reply:
I double checked, and this shouldn't break any existing signal handlers for this. 👍